Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3.9.9-buster as base-image
+
+RUN apt-get update && apt-get install -y --no-install-recommends apt-utils \
+    bash \
+    curl \
+    && mkdir project
```

Comment:
At least some of this is overkill - the `/project` folder will be automatically created by WORKDIR and the base image has bash & curl pre-installed. Can we get rid of all of this?

Reply:
Removed all!